hwilib/ledgeri.py

```diff
@@ -18,9 +18,7 @@ class LedgerClient(HardwareWalletClient):
     # hacked in device support using btchip-python
     def __init__(self, device):
         super(LedgerClient, self).__init__(device)
-        # hack to use btchip-python's getDongle pipeline
-        device.close()
-        self.dongle = getDongle(logging.getLogger().getEffectiveLevel() == logging.DEBUG)
+        self.dongle = HIDDongleHIDAPI(device, True, logging.getLogger().getEffectiveLevel() == logging.DEBUG)
```

Comment:
Is the second parameter always `True`? Looking at btchip-python, it seems that there are two device id pairs that have it set to `False`. Should we take those into account?

Reply:
I believe those are non-nano s, I'll look.

Reply:
https://github.com/greenaddress/GreenBits/blob/master/app/src/main/res/xml/device_filter.xml

Reply:
Right now we will accept non-nano s ledger devices.

Reply:
They use a completely different API, for many calls. I'd be shocked if anything worked.